### ISSUE TITLE
Can't click links to the line of code with the error

### ISSUE THREAD

**Issue by carlospisarello** [eslint/eslint#10409]
* **ESLint Version: ^8.2.1**
* **Node Version: 8.11.1**
* **npm Version: 5.6.0**

**What parser (default, Babel-ESLint, etc.) are you using?**
Babel-ESLint

**Please show your full configuration:**

```js
// https://eslint.org/docs/user-guide/configuring

module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint'
  },
  env: {
    browser: true,
  },
  // https://github.com/vuejs/eslint-plugin-vue#priority-a-essential-error-prevention
  // consider switching to `plugin:vue/strongly-recommended` or `plugin:vue/recommended` for stricter rules.
  extends: ['plugin:vue/essential', 'airbnb-base'],
  // required to lint *.vue files
  plugins: [
    'vue'
  ],
  // check if imports actually resolve
  settings: {
    'import/resolver': {
      webpack: {
        config: 'build/webpack.base.conf.js'
      }
    }
  },
  // add your custom rules here
  rules: {
    // don't require .vue extension when importing
    'import/extensions': ['error', 'always', {
      js: 'never',
      vue: 'never'
    }],
    // disallow reassignment of function parameters
    // disallow parameter object manipulation except for specific exclusions
    'no-param-reassign': ['error', {
      props: true,
      ignorePropertyModificationsFor: [
        'state', // for vuex state
        'acc', // for reduce accumulators
        'e' // for e.returnvalue
      ]
    }],
    // allow optionalDependencies
    'import/no-extraneous-dependencies': ['error', {
      optionalDependencies: ['test/unit/index.js']
    }],
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
    'linebreak-style': 0,
    'max-len': 0,
    'no-console': 0,
    'no-plusplus' : 0,
  }
}
```

**What did you do?**

I was working on a project with the previous version of ESLint and when the parser showed an error or warning, i was able to click on a link that sent me to the line of code with that error, but that's no longer the case since i updated.

**What did you expect to happen?**
To click on the warning or error link to go to the line of code with that error.

**What actually happened? Please include the actual, raw output from ESLint.**
Absolutely nothing, i'm no longer able to click that link.

**Comment by platinumazure** [comment]
Hi @carlospisarello, thanks for the issue.

I'm confused about a couple of things:

1. The ESLint CLI does not really support clicking on a link to view the line of code in error-- I assume you are using an editor which does provide this capability. What editor are you using?
1. We don't actually have an ESLint 8.x yet. Are you referring to the version of the ESLint integration in your editor, maybe?
1. Could you please show the actual, raw output from ESLint (or your editor integration)?
